dist/Dockerfile.e2e/Dockerfile

```diff
@@ -24,14 +52,14 @@ RUN yum update -y && \
 COPY --from=rust_builder /opt/cincinnati/bin/e2e /usr/bin/cincinnati-e2e-test
 COPY --from=rust_builder /opt/cincinnati/bin/prometheus_query /usr/bin/cincinnati-prometheus_query-test
 COPY --from=rust_builder /opt/cincinnati/bin/slo /usr/bin/cincinnati-e2e-slo
-COPY --from=rust_builder /go/src/github.com/openshift/cincinnati/hack/e2e.sh hack/
```

Comment:
Previous `RUN` needs `libgit2` to fix e2e and report valid commit message

Reply:
I was wrong - `libgit2` must be present in build container, not necessary in e2e/deploy images